Update .section flags to modern notation (old style only supported on SPARC)

diff '--exclude=.git' -ur l4/pkg/bootstrap/server/src/ARCH-arm64/crt0.S l4.works/pkg/bootstrap/server/src/ARCH-arm64/crt0.S
--- l4/pkg/bootstrap/server/src/ARCH-arm64/crt0.S	2025-05-15 08:08:52.070000000 +0200
+++ l4.works/pkg/bootstrap/server/src/ARCH-arm64/crt0.S	2025-05-15 07:57:32.566000000 +0200
@@ -16,7 +16,7 @@
 	msr	\sctlr_reg, \tmp
 .endm
 
-.section .text.init,#alloc,#execinstr
+.section .text.init,"ax"
 .type _start,#function
 .globl _start
 _start:
diff '--exclude=.git' -ur l4/pkg/l4re-core/crtn/lib/src/ARCH-arm64/crt0_x.S l4.works/pkg/l4re-core/crtn/lib/src/ARCH-arm64/crt0_x.S
--- l4/pkg/l4re-core/crtn/lib/src/ARCH-arm64/crt0_x.S	2025-05-15 08:08:48.803000000 +0200
+++ l4.works/pkg/l4re-core/crtn/lib/src/ARCH-arm64/crt0_x.S	2025-05-15 07:56:48.350000000 +0200
@@ -9,7 +9,7 @@
 
 /*#include <l4/sys/compiler.h>*/
 
-.section .text.init,#alloc,#execinstr
+.section .text.init,"ax"
 .type _start,#function
 .globl _start
 _start:
diff '--exclude=.git' -ur l4/pkg/l4re-core/sigma0/server/src/ARCH-arm64/crt0.S l4.works/pkg/l4re-core/sigma0/server/src/ARCH-arm64/crt0.S
--- l4/pkg/l4re-core/sigma0/server/src/ARCH-arm64/crt0.S	2025-05-15 08:08:49.357000000 +0200
+++ l4.works/pkg/l4re-core/sigma0/server/src/ARCH-arm64/crt0.S	2025-05-15 08:00:05.455000000 +0200
@@ -7,7 +7,7 @@
  * Please see the COPYING-GPL-2 file for details.
  */
 
-.section .text.init,#alloc,#execinstr
+.section .text.init,"ax"
 .type _start,#function
 .globl _start
 _start:
